### DISCUSSION TITLE
Parallax X offset

### DISCUSSION THREAD

**Discussion by TomasSestak** [pmndrs/react-spring#1311] [Support] [answered]
Hi,

is there a way to animate X position of ParallaxLayer instead of Y?

**Comment by tomdohnal** [comment]
You can create a `horizontal` Parallax to animated the X position but I'm not sure if that's what you're looking for...

**Comment by kindoflew** [comment, marked as answer]
You can set `horizontal={true}` on a `ParallaxLayer` even if `Parallax` is set to scroll vertically. (The opposite is also possible -- you can set `horizontal={false}` on a `ParallaxLayer` when `Parallax` has `horizontal={true}`.)

**Reply by TomasSestak** [reply]
Yep, i see it in the docs now, wasn't implemented in the time of writing the question. Thanks!
